velox/exec/HashProbe.cpp

```diff
@@ -283,6 +283,27 @@ void HashProbe::maybeSetupSpillInputReader(
   spillPartitionSet_.erase(iter);
 }
 
+void HashProbe::initializeResultIter() {
+  VELOX_CHECK_NOT_NULL(table_);
```

Comment:
Shall we check if resultIter_ is not null here? As we might call this multiple times in case of spill?
```
if (resultIter_ != nullptr) {
  return;
}
```